bin/dfx-canister-url

```diff
@@ -95,17 +92,9 @@ while ! test -e dfx.json; do
   }
 done
 
-# Some canisters have custom URLs, such as identity.ic0.app.
-if [[ "${DFX_URL_TYPE:-}" == "static" ]]; then
-  url="$(jq -r '.canisters[env.DFX_CANISTER].url[env.DFX_NETWORK] // ""' dfx.json)"
-else
-  url=""
-fi
-
-# Not found?
-# Each network also has a pattern for canister URLs of the form: https://CANISTER_ID.NETWORK_URL
+# Each network has a pattern for canister URLs of the form: https://CANISTER_ID.raw?.NETWORK_URL
```

Comment:
So the `?` is supposed to say `.raw` is optional? That wasn't clear to me.
It might be better to specify 2 full patterns, or put `.raw` in parentheses.

Reply:
I have provided two patterns.